hack/release

```diff
@@ -1,21 +1,10 @@
 #!/usr/bin/env bash
 
-TAG=$1
-OUT=$2
-
 . $(dirname $0)/util
 set -eu -o pipefail
 
 : ${PLATFORMS=linux/amd64}
-
-usage() {
-  echo "usage: ./hack/release <tag> <out>"
-  exit 1
-}
-
-if [ -z "$TAG" ] || [ -z "$OUT" ]; then
-  usage
-fi
+: ${EXPORT_LOCAL=release-out}
```

Comment:
Not very fond of this env. The name isn't very clear and setting dest path is kind-of expected to be a parameter for the release. Maybe leave it as positional arg and default to `./release-out`. Unless there is some well-known env that is used in these cases that I can't think of atm.